NetworkingTests/Resource/ResourceTests.swift

```diff
@@ -2,10 +2,10 @@ import XCTest
 @testable import Networking
 
 final class ResourceTests: XCTestCase {
-    private let emptyDecoding = ResourceDecodingExecution.background(EmptyDecoder())
+    private let emptyDecoding = EmptyDecoder()
```

Comment:
should this be renamed to emptyDecoder now?

Reply:
done